middleware/portal.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const moment = require('moment');
+
+/**
+ * Serves a holding/downtime page for the AFA portal if redirected here
+ */
+module.exports = (req, res, next) => {
+    const downtimeEndDate = moment('2018-05-29');
```

Comment:
This will change (there's an upcoming maintenance period in the next week or two but I'm trying to get ahead of it), but illustrates the premise.